products/statement-generator/src/components-layout/AppFooter.tsx

```diff
@@ -39,6 +39,11 @@ const useStyles = makeStyles(({ breakpoints, palette, spacing }) =>
       display: 'flex',
       flexDirection: 'column',
       padding: spacing(1),
+      marginLeft: '-16px',
+
+      [breakpoints.down(breakpoints.values.md)]: {
+        marginLeft: '-8px',
+      },
```

Comment:
Because the weird footer alignment is caused by padding, best practice would say to adjust the padding before implementing a negative margin.